COPYING

```diff
@@ -1,11 +1,24 @@
 Compression Library Plug-in for HDF5
-Copyright (c) 2017-2019, The HDF Group.
+Copyright (c) 2017, The HDF Group. 
 
 All rights reserved.
 
-NOTICE:
-All information contained herein is, and remains, the property of
-The HDF Group. The intellectual and technical concepts contained
-herein are proprietary to The HDF Group. Dissemination of this
-information or reproduction of this material is strictly forbidden
-unless prior written permission is obtained from The HDF Group.
+Redistribution and use in source and binary forms, with or without 
+modification, are permitted for any purpose (including commercial purposes) 
+provided that the following conditions are met:
+
+1. Redistributions of source code must retain the above copyright notice, 
+   this list of conditions, and the following disclaimer.
+
+2. Redistributions in binary form must reproduce the above copyright notice, 
+   this list of conditions, and the following disclaimer in the documentation 
+   and/or materials provided with the distribution.
+
+3. Neither the name of The HDF Group nor the name of any Contributor may be     used to endorse or promote products derived from this software without  specific prior written permission from The HDF Group or the Contributor,        respectively.
```

Comment:
weird split of line?

Reply:
tabs from word.  Also broke the long lines for < 80 characters to match the HDF5 copying file (mostly).